public/js/export.js

```diff
@@ -207,24 +215,26 @@ function getDataCreds(link) {
 }
 
 function getFeatureJson() {
-    //build our feature information
+    // build our feature information
     var feature = {};
     feature.featureKey = getExistingFeature();
     feature.featureTags = getFeatureTags();
     feature.featureLinks = getFeatureLinks();
     feature.featureTitle = getFeatureTitle();
     feature.featureDescription = getFeatureDescription();
+    feature.backgroundTitle = getBackgroundTitle();
+    feature.backgroundDescription = getBackgroundDescription();
     feature.backgroundSteps = getBackgroundTestSteps();
     return feature;
 }
 
 function getScenariosJson() {
-    //build our scenario information
+    // build our scenario information
     var scenarios = [];
     $('.scenario').each(function() {
         scenario = {};
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "scenario" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=javascript%3AS2703)